doc/api/perf_hooks.md

```diff
@@ -444,6 +444,31 @@ If the wrapped function returns a promise, a finally handler will be attached
 to the promise and the duration will be reported once the finally handler is
 invoked.
 
+### `performance.uvMetricsInfo()`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Returns: {Object}
+  * `loopCount` {number} Number of event loop iterations.
+  * `events` {number} Number of events that have been processed by the event handler.
+  * `events_waiting` {number} Number of events that were waiting to be processed when the event provider was called.
```

Comment:
Probably want this to be `eventsWaiting` or switch the other to `loop_count` for consistency.